app/components/treatment/voting/modal.js

```diff
@@ -33,18 +33,19 @@ export default class TreatmentVotingModalComponent extends Component {
   @restartableTask
   /** @type {import("ember-concurrency").Task} */
   fetchStemmingen = function* () {
-    this.stemmingen = yield this.args.behandeling.stemmingen;
+    this.stemmingen = (yield this.args.behandeling.stemmingen).sortBy('position');
   };
 
   @task
   /** @type {import("ember-concurrency").Task} */
   saveStemming = function* () {
     const isNew = this.editStemming.stemming.isNew;
-    yield this.editStemming.saveTask.perform();
     if (isNew) {
+      this.editStemming.stemming.position = this.args.behandeling.stemmingen.length;
       this.args.behandeling.stemmingen.pushObject(this.editStemming.stemming);
       this.args.behandeling.save();
     }
+    yield this.editStemming.saveTask.perform();
```

Comment:
I think this needs to be before the save of the behandeling, not a 100% sure

Reply:
I tested and it worked fine, but I can leave it before, I just changed it in order to be a little bit more concise because I need to put 2 if isNew statements if not, I will change it just to be sure